sokol_app.h

```diff
@@ -9609,7 +9613,11 @@ _SOKOL_PRIVATE void _sapp_x11_release_error_handler(void) {
 }
 
 _SOKOL_PRIVATE void _sapp_x11_init_extensions(void) {
-    _sapp.x11.UTF8_STRING             = XInternAtom(_sapp.x11.display, "UTF8_STRING", False);
+    if (_sapp.clipboard.enabled) {
+        _sapp.x11.UTF8_STRING         = XInternAtom(_sapp.x11.display, "UTF8_STRING", False);
+        _sapp.x11.CLIPBOARD           = XInternAtom(_sapp.x11.display, "CLIPBOARD", False);
+        _sapp.x11.TARGETS                = XInternAtom(_sapp.x11.display, "TARGETS", False);
```

Comment:
A minor thing but can you fix the indentation here?

Reply:
Done, thanks